CONTRIBUTING_CODE.md

```diff
@@ -107,7 +107,7 @@ Great Expectations requires a Python version from 3.8 to 3.11.
 
     To specify other dependencies, add a comma after `test` and enter the dependency name(s). **For example, ".[test, postgresql, trino]"**.
 
-    The supported extra dependencies include: `arrow`, `athena`, `aws_secrets`, `azure`, `azure_secrets`, `bigquery`, `clickhouse`, `cloud`, `dremio`, `excel`, `gcp`, `hive`, `mssql`, `mysql`, `pagerduty`, `postgresql`, `redshift`, `s3`, `snowflake`, `spark`, `teradata`, `test`, `trino`, `vertica`.
```

Comment:
We should leave this here since installing with `[cloud]` will be necessary to get the `orjson` dependency which is necessary to talk to mercury for metric storage according to @anthonyburdi.

Note, if you want to remove an extra, such as `[cloud]`, you will need to edit `setup.py`. In this case, you could add `"cloud"` to [ignore_keys](https://github.com/great-expectations/great_expectations/blob/develop/setup.py#L37).

Reply:
Ah, ok - thanks for that clarification. I re-added `cloud` in this list of supported extra dependencies.